tests/Utils.test.cpp

```diff
@@ -1,31 +1,51 @@
 #include "doctest.h"
 #include "LSP/Utils.hpp"
+#include "LSP/Sourcemap.hpp"
 
 TEST_SUITE_BEGIN("UtilsTest");
 
 TEST_CASE("getAncestorPath finds ancestor from given name")
 {
-    CHECK_EQ(getAncestorPath("game/ReplicatedStorage/Module/Child/Foo", "Module"), "game/ReplicatedStorage/Module");
+    auto sourceNode = std::make_shared<SourceNode>(SourceNode{});
```

Comment:
Rather than creating sourcenodes for all these tests, can we just pass `nullptr`?